xla/service/cpu/tests/onednn_convolution_test.cc

```diff
@@ -32,159 +33,221 @@ limitations under the License.
 namespace xla {
 namespace cpu {
 
-class ConvolutionTest : public HloTestBase {
+class ConvolutionTest : public HloTestBase,
+                        public ::testing::WithParamInterface<PrimitiveType> {
  protected:
   DebugOptions GetDebugOptionsForTest() override {
     DebugOptions debug_options = HloTestBase::GetDebugOptionsForTest();
     debug_options.set_xla_cpu_use_thunk_runtime(false);
     return debug_options;
   }
 
-  const char* conv_rewrite_str_ = R"(
-    ; CHECK:     custom_call_target="__onednn$convolution",
-    ; CHECK:       backend_config={
-    ; CHECK-DAG:     "outer_dimension_partitions":[],
-    ; CHECK-DAG:       "onednn_conv_config":{
-    ; CHECK-DAG:   }
-    ; CHECK:     }
-    )";
+  PrimitiveType dtype;
+  std::string dtypeString;
+  bool user_scratchpad;
+  bool weights_prepacked;
+  float atol;
+  float rtol;
```

Comment:
Nit: Our [C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Variable_Names) requires class members' names ending with underscore (`_`). Variable names should also be snake-case (lowercase + underscore).
```suggestion
  PrimitiveType dtype_;
  std::string dtype_string_;
  bool user_scratchpad_;
  bool weights_prepacked_;
  float atol_;
  float rtol_;
```

Reply:
Done